### ISSUE TITLE
(Android) (Burnout 3: Takedown (US)) Black Screen of Death

### ISSUE THREAD

**Issue by DiegoGTRatty** [jpd002/Play-#473]
https://scontent.fphx1-1.fna.fbcdn.net/v/t1.0-9/fr/cp0/e15/q65/1044005_677524369083304_3044425493400539960_n.jpg?efg=eyJpIjoidCJ9&oh=fc7f2987a34928ffa8dc53b571e1f2d1&oe=5870757B

First of all, yes, I'm aware that this game isn't compatible as of me posting this at this time.

Anyways, the game boots up but stays at the loading screen shown in the link. Since it doesn't crash, or not load, that indicates that it's sort of compatible, after a few (probably a lot but I like to stay positive) bug fixes within Play? Any thoughts?

**Comment by jpd002** [comment]
Yeah, it's not quite there yet. I've been looking at the issues with this game and got some fixes in progress, but it's still stuck on that loading screen even with those fixes.

**Comment by DiegoGTRatty** [comment]
Hmmm... Well that's interesting.

Thanks for the fast and direct response though. :)

**Comment by atomic83GitHub** [comment]
There is a compat report for this game, this should be close
